src/patternfly/demos/Card/card-content-3.hbs

```diff
@@ -0,0 +1,24 @@
+{{#> card-header}}
+  {{#> l-flex flex--modifier="pf-m-align-center"}}
+    {{#> l-flex flex--modifier="pf-m-grow"}}
```

Comment:
You can simplify this section quite a bit for the layouts in these demos.

```
<div class="pf-c-card__header pf-c-title pf-m-lg pf-l-flex pf-m-space-items-none pf-m-align-center">
    <img class="pf-m-align-left" src="/assets/images/camel-spark_200x150.png" width="60px" alt="Logo">
    <div class="pf-c-dropdown">...</div>
    <input type="checkbox" id="card-3-check" name="card-3-check" aria-labelledby="card-3-check-label">
</div>
```

Or you could add a bit more HTML and do it like

```
<div class="pf-c-card__header pf-c-title pf-m-lg">
    <div class="pf-l-flex pf-m-space-items-none pf-m-align-center">
        <img class="pf-m-align-left" src="/assets/images/camel-spark_200x150.png" width="60px" alt="Logo">
        <div class="pf-c-dropdown">...</div>
       <input type="checkbox" id="card-3-check" name="card-3-check" aria-labelledby="card-3-check-label">
    </div>
</div>
```